pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -718,8 +716,15 @@ def cmd_connection_save(context, name):
     connections = context.connections_repo
 
     context.spinner_stop()
-
-    connections.add(save_connection.name, save_connection)
+    try:
+        connections.add(save_connection)
+    except Exception as exc:
+        click.echo('Fatal error loading connections file: "{0}". '
+                   "Exception: {1}: {2}".
+                   format(connections.connections_file,
+                          exc.__class__.__name__, exc),
+                   err=True)
```

Comment:
The exception message that now comes up has everything to be displayed as is, without adding more context or exception names or connections file name (except for the `IOError` case, which is another reason to handle it in `add()`). So I suggest to just display it.